scripts/utilities/run_histmakers.sh

```diff
@@ -0,0 +1,45 @@
+# Due to memory constraints run the histmaker multiple times and append the output file gradually
```

Comment:
Is there a particular motivation to have a shell script rather than a python one? I know it seems more natural for this purpose, but python scripts are easier to deal with and can do more elaborated operations more easily

Reply:
No particular motivation.